<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="ChampIconArea">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>300</width>
      <height>750</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>298</width>
       <height>748</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>280</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>301</width>
        <height>701</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="ChampIconLayout" rowstretch="0" columnstretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="ChampInfoArea">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>480</width>
      <height>750</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>478</width>
       <height>748</height>
      </rect>
     </property>
     <widget class="QPushButton" name="TierSort">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>티어</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NameSort">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>이름</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WinrateSort">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>승률</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PickRateSort">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>픽률</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BanRateSort">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>밴율</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>60</y>
        <width>481</width>
        <height>691</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ChampInfoLayout">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../UIResource/champIcons.qrc"/>
 </resources>
 <connections/>
</ui>
